.github/workflows/canary-integration-test.yml

```diff
@@ -564,7 +564,6 @@ jobs:
         kubectl create -f tests/manifests/test-cluster-on-pvc-encrypted.yaml
         yq merge --inplace --arrays append tests/manifests/test-object.yaml tests/manifests/test-kms-vault-spec.yaml
         sed -i 's/ver1/ver2/g' tests/manifests/test-object.yaml
-        sed -i 's/VAULT_BACKEND: v1/VAULT_BACKEND: v2/g' tests/manifests/test-object.yaml
```

Comment:
We may need to add this manually `VAULT_BACKEND: v2` for `test-object.yaml`. Otherwise test case will fail since RGW supports only v2. Other more sensible/clean option since it is auto-detecting for OSD encryption, for RGW confs if kv engine is specified then it will be v2 only instead of checking the possibility of v1. This requires a bit more code changes in `CheckRGWKMS()`

Reply:
Ok will look, thanks!

Reply:
Not resolved I guess, if it is not planned for this PR. I can take it up as separate PR

Reply:
Why not resolved?

Reply:
@leseb: please refer https://github.com/rook/rook/pull/8265/files#r673680345